testing/btest/language/skip-opt1.zeek

```diff
@@ -0,0 +1,24 @@
+# @TEST-EXEC: zeek -b %INPUT >out 2>&1
```

Comment:
Can these three tests be combined into one that runs `zeek` three times? It's the same script code, so it seems weird to duplicate it.

Reply:
I didn't want to do that for these because then the `-O gen-C++` maintenance script won't be able to run on them, since it can't handle combined Zeek runs (because it uses the Zeek run in the script to drive the C++ generation).